src/Confluent.Kafka/Timestamp.cs

```diff
@@ -80,6 +88,15 @@ public static long DateTimeToUnixTimestampMs(DateTime dateTime)
             }
         }
 
+        /// <summary>
+        ///     Convert a unix timestamp to a DateTime value.
+        /// </summary>
+        /// <param name="timestamp">
+        ///     The unix timestamp to convert.
+        /// </param>
+        /// <returns>
+        ///     The DateTime value associated with <paramref name="timestamp"/>
+        /// </returns>
         public static DateTime UnixTimestampMsToDateTime(long timestamp)
             => new DateTime(1970, 1, 1, 0, 0, 0, 0, System.DateTimeKind.Unspecified)
```

Comment:
Not doc related, bu this should be Utc kind - UnixTimestamp is Utc, not Unspecified.
Also i saw that producer use timestamp?.Ticks instead of DateTimeToUnixTimestampMs(datetime) in Produce, will post an issue/PR related to this tomorrow